src/models/Image.js

```diff
@@ -6,4 +6,5 @@ export type Image = {
   url: string,
   tags: Array<Tag>,
   score: number,
+  delete?: Function,
```

Comment:
This is okay for now but we should consider making decorated types for stuff like this.